keyboards/fallacy/info.json

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
One last thing, I think - the layouts here are a bit wonky. You can preview the info.json for testing purposes in the Configurator by pressing Ctrl+Shift+I.
![image](https://user-images.githubusercontent.com/4781841/85911859-419c5f80-b86b-11ea-848c-14b03d874513.png)

Reply:
Sorry about that. I was wondering if this layout was going to work - I checked the QMK documentation and it stated that rotation was supported, but I was a bit unsure if it actually would or not as I had never seen it on configurator before. I've gone ahead and removed any rotation now.